test/allowlist_for_publicAPI.json

```diff
@@ -1300,5 +1300,1098 @@
+    "RendezvousTimeoutError",
+    "cas_delay",
+    "parse_rendezvous_endpoint"
+  ],
+  "torch.distributed.elastic.rendezvous.etcd_rendezvous_backend": [
+    "EtcdAlreadyExist",
+    "EtcdClient",
+    "EtcdCompareFailed",
+    "EtcdException",
+    "EtcdKeyNotFound",
+    "EtcdResult",
+    "EtcdStore",
+    "Optional",
+    "RendezvousBackend",
+    "RendezvousConnectionError",
+    "RendezvousParameters",
+    "RendezvousStateError",
+    "Store",
+    "Token",
+    "Tuple",
+    "b64decode",
+    "b64encode",
+    "cast",
+    "parse_rendezvous_endpoint"
+  ],
+  "torch.distributed.elastic.rendezvous.etcd_server": [
+    "Optional",
+    "TextIO",
+    "Union"
+  ],
+  "torch.distributed.elastic.rendezvous.etcd_store": [
+    "Optional",
+    "Store",
+    "b64decode",
+    "b64encode"
+  ],
+  "torch.distributed.elastic.rendezvous.static_tcp_rendezvous": [
+    "Optional",
+    "PrefixStore",
+    "RendezvousHandler",
+    "RendezvousParameters",
+    "Store",
+    "TCPStore",
+    "Tuple",
+    "cast",
+    "parse_rendezvous_endpoint"
+  ],
+  "torch.distributed.elastic.utils.distributed": [
+    "closing",
+    "get_logger"
+  ],
+  "torch.distributed.fsdp.sharded_grad_scaler": [
+    "Any",
+    "Dict",
+    "GradScaler",
+    "Iterable",
+    "List",
+    "OptState",
+    "Optional",
+    "ProcessGroup",
+    "Sequence",
+    "Tuple",
+    "Union",
+    "defaultdict",
+    "overload"
+  ],
+  "torch.distributed.launch": [
+    "get_args_parser",
+    "run"
+  ],
+  "torch.distributed.pipeline.sync": [
+    "NoChunk",
+    "WithDevice"
+  ],
+  "torch.distributed.rpc.rref_proxy": [
+    "Future",
+    "partial",
+    "rpc_async"
+  ],
+  "torch.distributed.run": [
+    "ArgumentParser",
+    "Callable",
+    "LaunchConfig",
+    "List",
+    "Std",
+    "Tuple",
+    "Union",
+    "check_env",
+    "elastic_launch",
+    "env",
+    "get_logger",
+    "macros",
+    "record"
+  ],
+  "torch.fx.annotate": [
+    "Proxy",
+    "compatibility"
+  ],
+  "torch.fx.experimental.accelerator_partitioner": [
+    "Deque",
+    "Device",
+    "Dict",
+    "GraphModule",
+    "List",
+    "NamedTuple",
+    "Node",
+    "NodeLatency",
+    "Partition",
+    "PartitionMode",
+    "PartitionerConfig",
+    "Set",
+    "Tuple",
+    "deque",
+    "get_extra_size_of",
+    "get_latency_of_partitioned_graph",
+    "get_partition_to_latency_mapping",
+    "get_size_of_all_nodes",
+    "map_arg",
+    "split_module"
+  ],
+  "torch.fx.experimental.graph_gradual_typechecker": [
+    "BatchNorm2d",
+    "Callable",
+    "Conv2d",
+    "Dict",
+    "Equality",
+    "Node",
+    "Target",
+    "TensorType",
+    "Var",
+    "is_consistent",
+    "is_more_precise",
+    "reduce"
+  ],
+  "torch.fx.experimental.merge_matmul": [
+    "Dict",
+    "List",
+    "Node",
+    "Tuple",
+    "legalize_graph",
+    "symbolic_trace"
+  ],
+  "torch.fx.experimental.meta_tracer": [
+    "Any",
+    "Callable",
+    "Dict",
+    "Optional",
+    "Union"
+  ],
```

Comment:
Uh, that's why people use `__all__`...

Reply:
Yes, a bunch of these can be fixed by adding `__all__`.
I didn't to keep this PR "simple" (just generate the json). But a bunch of these should be easy to fix.

Reply:
Yeah I wasn't saying that you should do it, I was more saying this about the folks who wrote these files haha